### PR TITLE
[Improvement][ui] Introduce remixicon, delete ans-ui license and font-awesome

### PR DESCRIPTION
**Pull request by break60**
Supplement the front-end dependency license document

### CODE REVIEW

**Comment by xingchun-chen** [comment]
+1

File: dolphinscheduler-dist/release-docs/LICENSE [hunk 1]
```diff
@@ -508,9 +507,14 @@ MIT licenses
...
+    remixicon 2.5.0 https://github.com/Remix-Design/remixicon   Apache-2.0
 
 
 ========================================
 BSD licenses
 ========================================
     d3 3.5.17: https://github.com/d3/d3 BSD-3-Clause
+
+========================================
+OFL licenses
```
Comment (chengshiwen) [inline dolphinscheduler-dist/release-docs/LICENSE:519]: These lines should be removed.

Comment (break60) [reply dolphinscheduler-dist/release-docs/LICENSE]: > These lines should be removed.
Deleted

**Review by chengshiwen — APPROVED** [review]

**Comment by chengshiwen** [comment]
+1

**Review by lgcareer — APPROVED** [review]
LICENSE +1

**Pull request merged**